index.bs

```diff
@@ -9824,6 +9824,63 @@ Methods</h5>
 		</div>
 </dl>
 
+<h5 id="AudioWorkletProcessor-instantiation">
+The instantiation of AudioWorkletProcessor</h5>
+
+The construction of an {{AudioWorkletNode}} is followed by the construction
+of a corresponding {{AudioWorkletProcessor}} instance in the
+{{AudioWorkletGlobalScope}}.
+
+At the end of the {{AudioWorkletNode}} construction,
+<dfn>processor construction data</dfn> will be prepared for the cross-thread
```

Comment:
This definition of "processor construction data" is normative, though.  Maybe this section should in fact be normative, remove the informative bits, and do the following:

1) Define what a "processor construction data" is.  
2) Define an algorithm like "create an AudioWorkletProcessor from processor construction data" which defines the steps to do that.

Then the step at the end of the AudioWorkletNode constructor will do something like "queue a control message to create an AudioWorkletProcessor from processor construction data consisting of nodeName as the nodeName, serializedProcessorPort as the nodePort, and serializedOptions as the nodeOptions" or so.  Might be worth looking in the infra spec to see how best to define the construction data and describe its creation.

Reply:
I think both points are addressed by this PR. I just need to be more explicit about an AWP being created from the construction data. I'll address this over next few commits.

Reply:
Can you point where I can find such example? I looked https://infra.spec.whatwg.org/, but couldn't find examples that uses the similar pattern.

Reply:
The infra spec doesn't have existing "construction data" things.  But it does define https://infra.spec.whatwg.org/#structs which is basically what you are using here.  So you would say that the "construction data" is a [struct](https://infra.spec.whatwg.org/#structs) and the items in it have the names you want, etc.  This isn't strictly required, but it makes it clear that this "construction data" is a notational convenience spec data structure.

Reply:
Forming a struct sounds reasonable, but I am not sure if that'll solve any real problem because such struct will not be consumed by any exposed interface.

Reply:
Infra structs are _never_ consumed by anything exposed.  They are just agreed-on ways of bundling several items together into a single package in a spec, so people reading the spec know what the terminology being used means.

Reply:
Sure. As I mentioned in the comment below, the processor construction data issue will be addressed in a different PR.